packages/common/src/store/pages/chat/selectors.ts

```diff
@@ -150,3 +151,62 @@ export const getUnfurlMetadata = (
   const message = getChatMessageById(state, chatId, messageId)
   return message?.unfurlMetadata
 }
+
+export const getUserChatPermissions = createSelector(
```

Comment:
fyi: @dharit-tan breaking change for mobile potentially

Reply:
Love this